Foundations/Pages/Profile.swift

```diff
@@ -0,0 +1,128 @@
+                    Text("Me sentindo... atarefada")
+                    Button(action: {
+                        
+                    }) {
+                        Text("Novo Card")
+                            .padding(.vertical, 5)
+                            .foregroundColor(.gray)
+                            .frame(maxWidth: .infinity,
+                                   alignment: .leading)
+                    }.buttonStyle(.bordered)
+                        .padding(.horizontal, 30)
+                    LazyVGrid(
+                        columns: columns,
+                        spacing: 20
+                    ) {
+                        ProfileCard(
+                            title: "Histórico"
+                        ) {
+                            CustomText("Confira as dinâmicas que você concluiu")
+                        }
+                        ProfileCard(
+                            title: "Conexões"
+                        ) {
+                            HStack {
+                                ProfileCircle("profile",
+                                              radius: 45)
+                                    .frame(height: 25)
+                                ProfileCircle("kiev-gama",
+                                              radius: 45)
+                                    .padding(.leading, -20)
+                                    .frame(height: 25)
+                                Text("...").font(.custom(
+                                        "Inter-Light",
+                                        size: 16
+                                ))
+                            }
+                        }
+                        ProfileCard(
+                            title: "Feedbacks"
+                        ) {
+                            CustomText("Leia os feedbacks que você recebeu")
+                        }
+                        ProfileCard(
+                            title: "Músicas"
+                        ) {
+                            CustomText("Confira as músicas que você está compartilha")
+                        }
+                        ProfileCard(
+                            title: "ScrollView Teste"
+                        ) {
+                            CustomText("Testando o scrollView")
+                        }
+                        ProfileCard(
+                            title: "ScrollView Teste"
+                        ) {
+                            CustomText("Testando o scrollView")
+                        }
+                        ProfileCard(
+                            title: "ScrollView Teste"
+                        ) {
+                            CustomText("Testando o scrollView")
+                        }
+                        ProfileCard(
+                            title: "ScrollView Teste"
+                        ) {
+                            CustomText("Testando o scrollView")
+                        }
```

Comment:
Is this the best approach to do this?
I was wondering whether we could take on a bit more automatized and optimized approach, even because it seems like there are lots of scrollables